stackbit.config.ts

```diff
@@ -10,34 +13,33 @@ import { Link } from './.stackbit/models/Link';
 import { Page } from './.stackbit/models/Page';
 import { ThemeStyle } from './.stackbit/models/ThemeStyle';
 
-const sbConfig = {
-    stackbitVersion: '~0.5.0',
-    ssgName: 'nextjs',
-    cmsName: 'git',
-    nodeVersion: '16',
-    dataDir: 'content/data',
-    pagesDir: 'content/pages',
-    pageLayoutKey: 'type',
-    styleObjectModelName: 'ThemeStyle',
-    assets: {
+const gitContentSource = new GitContentSource({
+    rootPath: __dirname,
+    contentDirs: ['content'],
+    models: [Button, Card, CardsSection, Config, Footer, Header, HeroSection, Image, Link, Page, ThemeStyle],
```

Comment:
Please put all models in an array in a separate file (as done in content-ops) so you can also remove all model imports here and replace them with a single one.

Reply:
updated, please check @eladroz.